flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/AbstractCheckpointStats.java

```diff
@@ -93,6 +93,11 @@ public abstract class AbstractCheckpointStats implements Serializable {
      */
     public abstract long getCheckpointedSize();
 
+    /** @return The metadata file size, 0 if unknown. */
+    public long getMetadataSize() {
+        return 0;
```

Comment:
As this is an abstract class, why not just skip the return?

Reply:
Right, neither `PendingCheckpointStats` nor `FailedCheckpointStats` has this value defined. I agree it's clearer to keep it here as abstracted and let `PendingCheckpointStats` implement it (which `FailedCheckpointStats` will inherit).

I'm also wondering if we do put this in the abstract class at all, and just expose in the `CompletedCheckpointStats`. Let me check this again.